include/aspect/material_model/visco_plastic.h

```diff
@@ -313,6 +313,8 @@ namespace aspect
         std::vector<double> angles_internal_friction;
         std::vector<double> cohesions;
         std::vector<double> exponents_stress_limiter;
+        
+         double max_yield_strength; // limit maximum yield stress from drucker-prager
```

Comment:
Please put the comment in the doxygen format above the variable declaration. That way it shows up in our online documentation at: https://aspect.geodynamics.org/doc/doxygen/classaspect_1_1MaterialModel_1_1ViscoPlastic.html

The format is as above:
```
/**
 * Comments
 */
double max_yield_strength;